share/python/driver.py

```diff
@@ -442,7 +442,7 @@ def energy(name, **kwargs):
     wfn = procedures['energy'][lowername](lowername, molecule=molecule, **kwargs)
 
     for postcallback in hooks['energy']['post']:
-        postcallback(lowername, **kwargs)
+        postcallback(lowername, wfn=wfn, **kwargs)
 
```

Comment:
I was thinking of this chunk of code C side in the SCF module. I guess its fine for now, but id like too look into this a bit more.

A few lines below this we have `lowername in ["EFP", ...`, this will not match of course. Need to lower those matches.